src/modules/Game.class.js

```diff
@@ -1,68 +1,66 @@
-'use strict';
+import Game from './modules/Game.class.js';
```

Comment:
The Game class should be declared here - in this file

Reply:
But as I see you've deleted it

You can copy the initial code here:
https://github.com/mate-academy/js_2048_game/blob/master/src/modules/Game.class.js

Then you should implement all methods of the class that are necessary to make it work